insights/headers/reply_to_mismatch_new.yml

```diff
@@ -0,0 +1,11 @@
+name: "New reply-to domain (<=30d) and sender mismatch"
+type: "query"
+source: |
+  map(filter(headers.reply_to,
+    length(headers.reply_to) > 0
```

Comment:
this check doesn't do anything for the filter

Reply:
Roger that will pull and validate! thx

Reply:
Tested worked great.